src/install/repository.zig

```diff
@@ -238,7 +238,7 @@ pub const Repository = extern struct {
 
         if (strings.hasPrefixComptime(url, "ssh://")) {
             final_path_buf[0.."https".len].* = "https".*;
-            bun.copy(u8, final_path_buf["https://".len..], url["ssh".len..]);
```

Comment:
2.5 hours 🫠 @dylan-conway @Jarred-Sumner this should fix the complex workspace tests.